FBSimulatorControl/Interactions/FBSimulatorInteraction.m

```diff
@@ -60,59 +60,42 @@ - (instancetype)copyWithZone:(NSZone *)zone
 
 #pragma mark Private
 
-- (instancetype)interactWithSimulator:(BOOL (^)(NSError **error, FBSimulator *simulator))block
+- (instancetype)interactWithSimulator:(BOOL (^)(id interaction, NSError **error, FBSimulator *simulator))block
```

Comment:
I think it would be better to actually define all interactions as `id<FBInteraction>` instead of `id`. I know it is not needed most of the time, but you lose compiler type checking for no reason.